<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="56102" started-at="2018-02-18T21:35:12Z" finished-at="2018-02-18T21:36:08Z">
    <groups>
    </groups>
    <test name="EcommerceTest" duration-ms="56102" started-at="2018-02-18T21:35:12Z" finished-at="2018-02-18T21:36:08Z">
      <class name="com.bjss.tests.EcommerceTest">
        <test-method status="PASS" signature="SetUp()[pri:0, instance:com.bjss.tests.EcommerceTest@43738a82]" name="SetUp" is-config="true" duration-ms="15320" started-at="2018-02-18T21:35:12Z" finished-at="2018-02-18T21:35:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="PASS" signature="raiseOrderTest()[pri:0, instance:com.bjss.tests.EcommerceTest@43738a82]" name="raiseOrderTest" duration-ms="26056" started-at="2018-02-18T21:35:27Z" finished-at="2018-02-18T21:35:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- raiseOrderTest -->
        <test-method status="PASS" signature="reviewOrderTest()[pri:1, instance:com.bjss.tests.EcommerceTest@43738a82]" name="reviewOrderTest" duration-ms="9107" started-at="2018-02-18T21:35:53Z" finished-at="2018-02-18T21:36:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reviewOrderTest -->
        <test-method status="FAIL" signature="captureFailureTest()[pri:2, instance:com.bjss.tests.EcommerceTest@43738a82]" name="captureFailureTest" duration-ms="4306" started-at="2018-02-18T21:36:02Z" finished-at="2018-02-18T21:36:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Yellow] but found [Maroon]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Yellow] but found [Maroon]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bjss.tests.EcommerceTest.captureFailureTest(EcommerceTest.java:135)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- captureFailureTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.bjss.tests.EcommerceTest@43738a82]" name="tearDown" is-config="true" duration-ms="376" started-at="2018-02-18T21:36:07Z" finished-at="2018-02-18T21:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.bjss.tests.EcommerceTest -->
    </test> <!-- EcommerceTest -->
  </suite> <!-- Suite1 -->
</testng-results>
